### PR TITLE
Add initiatives

### LINKED ISSUE #59: META - Initiatives

### PR DESCRIPTION
**Pull request by Fujistone**
closes #59 

See request here : https://metabase.osp.cat/model/1815-initiatives

### CODE REVIEW

File: initiatives/info.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+resource: budgets
```
Comment (Dynnammo) [inline initiatives/info.yml:2]: rename to `initiatives`

File: initiatives/info.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+query:
+  model: true
+  sql: |
+    select 
+        decidim_initiatives.id,
+        decidim_initiatives.id::text as id_as_text,
+        title->>'fr' as title,
```
Comment (Dynnammo) [inline initiatives/info.yml:9]: change `fr` to `$LANGUAGE_CODE`, as we're going to update the `decidim_metabase` repo to adapt to language of the requested database

File: initiatives/info.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        regexp_replace(decidim_initiatives.description->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as description,
```
Comment (Dynnammo) [inline initiatives/info.yml:10]: same as above, switch `fr` to `$LANGUAGE_CODE`

File: initiatives/info.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          when 2 then 'Supprimée'
+          when 3 then 'Publiée'
+          when 4 then 'Rejetée'
+          when 5 then 'Acceptée'
+        end) as parsed_state,
+        state,
+        signature_type,
+        signature_start_date,
+        signature_end_date,
+        regexp_replace(answer->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as answer,
```
Comment (Dynnammo) [inline initiatives/info.yml:25]: switch to `$LANGUAGE_CODE`

File: initiatives/info.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        parsed_online_votes,
+        parsed_offline_votes,
+        parsed_online_votes + parsed_offline_votes as sum_votes,
+        comments_count,
+        follows_count,
+        concat('https://', (select host from {{#organizations}} org), '/initiatives/i-',decidim_initiatives.id) as url,
+        supports_required,
+        decidim_areas.id as area_id,
+        (case when decidim_areas.name ? 'fr' then decidim_areas.name->>'fr' else 'Pas de catégorie' end) as area_name,
+        (case when decidim_area_types.name ? 'fr' then decidim_area_types.name->>'fr' else 'Pas de type de catégorie' end) as area_type_name
```
Comment (Dynnammo) [inline initiatives/info.yml:43]: switch to `$LANGUAGE_CODE`

File: initiatives/info.yml (hunk 1 above)
Comment (Fujistone) [reply initiatives/info.yml]: fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)

File: initiatives/info.yml (hunk 2 above)
Comment (Fujistone) [reply initiatives/info.yml]: fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)

File: initiatives/info.yml (hunk 3 above)
Comment (Fujistone) [reply initiatives/info.yml]: fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)

File: initiatives/info.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        regexp_replace(decidim_initiatives.description->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as description,
+        decidim_author_id,
+        published_at,
+        (case state
+          when 0 then 'Créée'
+          when 1 then 'Validée'
+          when 2 then 'Supprimée'
+          when 3 then 'Publiée'
+          when 4 then 'Rejetée'
+          when 5 then 'Acceptée'
```
[the remark this answers is not in the document]
Comment (Fujistone) [reply initiatives/info.yml]: changed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1). We switched backed to English in order to be more inclusive

File: initiatives/info.yml (hunk 4 above)
Comment (Fujistone) [reply initiatives/info.yml]: fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)

File: initiatives/info.yml (hunk 5 above)
Comment (Fujistone) [reply initiatives/info.yml]: fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)

**Review by Dynnammo — APPROVED** [review]

**Review by simonaszilinskas — APPROVED** [review]

**Pull request merged**